classes/person.rb

```diff
@@ -27,4 +28,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+    rental.book.rentals.push(rental) unless rental.book.rentals.include?(rental)
+  end
```

Comment:
- [ ] Instead of sending the rental as a parameter and setting the person as `self` afterward. Kindly use the following method of adding a rental to achieve the 'has-many' association.

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```